test/integers.test.ts

```diff
@@ -57,6 +57,12 @@ describe('Integer tests', () => {
 
   it('finds that adding 1000 makes any number larger and shrinks the example', () => {
     expect(fc.scenario()
+      .config(fc.strategy()
```

Comment:
Any reason why you added this here? If not, let's keep the PR clean and focused on a single thing.

Reply:
Yes to make sure the `biased` values are not being considered. This way we ensure that the shrinking process is working as intended.